schema.sql

```diff
@@ -0,0 +1,23 @@
+-- Create table genre
+DROP TABLE IF EXISTS "genre";
+CREATE TABLE "genre"(
+    "id" INTEGER NOT NULL,
+    "name" VARCHAR(255) NOT NULL
+);
+ALTER TABLE
+    "genre" ADD PRIMARY KEY("id");
+
+-- Create table music_album
+DROP TABLE IF EXISTS  "music_album";
+CREATE TABLE "music_album"(
+    "id" INTEGER NOT NULL,
```

Comment:
CREATE TABLE  music_album(
     id  INTEGER NOT NULL,
     genre  INTEGER NOT NULL FOREIGN KEY REFERENCES  genre(id),
     publish_date  DATE NOT NULL,
     archived BOOLEAN NOT NULL,
     on_spotify BOOLEAN NOT NULL
 
   PRIMARY KEY(id)
 );

Reply:
Hi @richardoppiyo ,
I updated the statement to include the constraint in the same create statement:
CREATE TABLE music_album(
    id INTEGER NOT NULL PRIMARY KEY,
    genre INTEGER NOT NULL,
    publish_date DATE NOT NULL,
    archived BOOLEAN NOT NULL,
    on_spotify BOOLEAN NOT NULL,
    FOREIGN KEY (genre)
        REFERENCES genre (id)
);